test/cluster-tests/dcos/test.sh

```diff
@@ -64,6 +68,8 @@ while [[ ${count} -lt 25 ]]; do
   if [[ "${running}" == "3" ]]; then
     log "Found 3 running tasks"
     break
+  else
+    ${remote_exec} ./dcos marathon app list
```

Comment:
you will be printing it every cycle?
We already have this here:

if [[ "${running}" != "3" ]]; then
  log "marathon validation failed"
  ${remote_exec} ./dcos marathon app show /web
  exit 1
fi